src/lib/openapi/spec/tag-schema.ts

```diff
@@ -11,16 +11,16 @@ export const tagSchema = {
     properties: {
         value: {
             type: 'string',
-            minLength: TAG_MIN_LENGTH,
-            maxLength: TAG_MAX_LENGTH,
-            description: 'The value of the tag',
+            pattern: `^\\s*\\S.{${TAG_MIN_LENGTH - 2},${
```

Comment:
I'm assuming it has no impact on the orval type

Reply:
No, it shouldn't I don't think Orval cares about regexes because they're hard to encode in typescript. The description has changed, though, so we should also regenerate Orval for that.